WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift

```diff
@@ -1385,6 +1385,7 @@ extension WooAnalyticsEvent {
             static let millisecondsSinceCardCollectPaymentFlow = "milliseconds_since_card_collect_payment_flow"
             static let siteID = "site_id"
             static let connectionType = "connection_type"
+            static let receiptSource = "source"
```

Comment:
regarding all the event names, parameter names, etc., is there a place where we keep documentation of all of it?

Reply:
There was an spreadsheet, but this has been deprecated since was too much manual work and prone to error. At the moment we just see them under MC > Tracks > Events. There if you look for `woocommerceios_` you'll see all registered events and their properties.

Some context: p91TBi-aNE-p2